### DISCUSSION TITLE
When I use docker to do harvester, the config.yaml file will be reset when docker restarts

### DISCUSSION THREAD

**Discussion by guitengyue** [Chia-Network/chia-blockchain#17605] [Support]
When I use docker to do harvester, the config.yaml file will be reset when docker restarts because compressed plot is used, parallel_decompressor_count: 2
decompressor_thread_count: 4. After I manually modified these two parameters, they can only be achieved through the command in the container: chia start harvester -r. When I restart the container, these two values ​​will automatically become 0.

**Comment by BrandtH22** [comment]
Hey @guitengyue , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia)

Make sure that the config file for the docker instance is persistent and piped in via the -v flag when launching the docker instance (should be similar for the db, plots, and keyring):
https://github.com/Chia-Network/chia-docker?tab=readme-ov-file#persist-configuration-db-and-keyring

Let us know if any questions arise :+1:

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
